src/tasks/templates.js

```diff
@@ -3,16 +3,27 @@ const { copyFiles } = require('mrm-core');
 
 // These files will be overwritten without any confirmation
 const files = [
-  '.travis.yml',
+  '.circleci/config.yml',
   'LICENSE',
   '.github/ISSUE_TEMPLATE.md',
   '.github/PULL_REQUEST_TEMPLATE.md',
+  '.prettierrc',
+  'appveyor.yml',
+  'eslint.js',
 ];
 
 // These files will by created only once
 const filesOnce = [
   'src/index.js',
   'src/cjs.js',
+  'src/options.json',
+  'test/errors.test.js',
+  'test/loader.test.js',
+  'test/options.test.js',
```

Comment:
`test/options.test.js` => `test/options/option.test.js`

```
test
|– [e2e]
|– fixtures
|– helpers
|– options
|
|– Errors.test.js
|– loader.test.js
```

Reply:
```
e2e
|– default.test.js
|– option.test.js
|– option1.test.js
```

**default.test.js** (Example)
```js
import path from 'path';
import dom from '../helpers/dom';
import webpack from '../helpers/compiler';

describe('E2E', () => {
  test('HTML', async () => {
    const config = {
      rules: [
        {
          test: /\.html$/,
          use: [
            {
              loader: path.resolve('./src'),
              options: {}
            }
          ]
        },
        {
          test: /\.png$/,
          use: [ 'file-loader' ]
        }
      ]
    };

    const stats = await webpack('e2e/html.js', config);
    const { assets } = stats.compilation;

    const scripts = {
      main: assets['main.js'].source(),
      runtime: assets['runtime.js'].source()
    };

    const { window } = dom([ scripts.runtime, scripts.main ]);

    expect(window.document.body.innerHTML).toMatchSnapshot();
  })
})
```

**helpers/dom.js** (JSDOM)
```js
import { JSDOM, VirtualConsole } from 'jsdom';

const html = (scripts) => `
<!DOCTYPE html>
<html>
  <head>
    <script>${scripts[0]}</script>
    <script>${scripts[1]}</script>
  </head>
  <body>
    <div id="root"></div>
  </body>
</html>`;

const virtualConsole = new VirtualConsole().sendTo(console)

export default function (scripts) {
  const dom = new JSDOM(
    html(scripts),
    { runScripts: "dangerously", virtualConsole }
  );

  return dom;
};
```

Maybe e2e isn't needed by default, needs triage/exploration with other loaders besides `style-loader, html-loader` e.g `imports-loader, exports-loader, expose-loader` and the difficulty to apply them :)

Reply:
I didn't want to go overboard with the minimal test setup. People can always add folders to contain stuff as a suite grows.

Reply:
It's your test setup, update this as you see fit. Just commit the changes directly to the `next` branch.

Reply:
> I didn't want to go overboard with the minimal test setup. People can always add folders to contain stuff as a suite grows.

Yes better to leave this out for now, I can reference additional helpers like `helpers/dom.js` in `test/README` in case someone needs them. This is also a bit in flux atm and neither required nor urgent at all

> It's your test setup, update this as you see fit. Just commit the changes directly to the next branch.

👍